proxy/http/HttpTransact.cc

```diff
@@ -8199,13 +8199,15 @@ HttpTransact::build_error_response(State *s, HTTPStatus status_code, const char
     if (has_ua_msie)
       s->hdr_info.client_response.value_set(MIME_FIELD_PROXY_CONNECTION, MIME_LEN_PROXY_CONNECTION, "close", 5);
   }
-  // Add a bunch of headers to make sure that caches between
-  // the Traffic Server and the client do not cache the error
-  // page.
+  // Make sure that caches between the Traffic Server and the client do not cache the error.
+  // ToDo: Not sure this matches expectation in RFC 7223.
```

Comment:
Well, I'm open to suggestion here. It seems, as per the RFC, that we probably shouldn't modify the Cache-Control header here? That seems inline with the code build_response() function too, right ?